### PR TITLE
Lecture 5 corrections (and possible correction to Lecture 3)

### PR DESCRIPTION
**Pull request by EvZissel**
In Bellman-Ford algorithm - policy update step

### CODE REVIEW

File: Lecture3.tex [hunk 1]
```diff
@@ -153,7 +153,7 @@ \subsection{The Bellman-Ford Algorithm}
 
       \tab{if  $d[v] < \infty $}
 
-      \tab{\tab{set $\pi [v] \in \arg {\min _u}\{ w(v,u) + \tilde d[u]:(v,u) \in E\} $}}
+      \tab{\tab{set $\pi [v] \in \arg {\min _u}\{ w(v,u) + d[u]:(v,u) \in E\} $}}
```
Comment (avivt) [inline Lecture3.tex]: This is actually OK as it is

File: Lecture5.tex [hunk 2]
```diff
@@ -122,8 +122,8 @@ \section{The Fixed-Policy Value Function}\label{s:FP_VF}
 \begin{itemize}
   \item The proof provides an explicit bound on $|{V^\pi }(s) - {V_n}{\kern 1pt} (s)|$. It may be seen that the convergence is exponential, with rate $O({\gamma ^n})$.
   \item Using vector notation, it may be seen that
-          \[{V_n}{\kern 1pt}  = {r^\pi } + {P^\pi }{r^\pi } +  \ldots  + {({P^\pi })^{n - 1}}{r^\pi } + {({P^\pi })^n}{V_0} = \sum\limits_{t = 0}^{n - 1} {{{({P^\pi })}^t}{r^\pi }}  + {({P^\pi })^n}{V_0}.\]
-Similarly,    ${V^\pi } = \sum\limits_{t = 0}^\infty  {{{({P^\pi })}^t}{r^\pi }}$.
+          \[{V_n}{\kern 1pt}  = {r^\pi } + {\gamma}{P^\pi }{r^\pi } +  \ldots  + {({\gamma}{P^\pi })^{n - 1}}{r^\pi } + {({\gamma}{P^\pi })^n}{V_0} = \sum\limits_{t = 0}^{n - 1} {{{({\gamma}{P^\pi })}^t}{r^\pi }}  + {({\gamma}{P^\pi })^n}{V_0}.\]
+Similarly,    ${V^\pi } = \sum\limits_{t = 0}^\infty  {{{({\gamma}{P^\pi })}^t}{r^\pi }}$.
```
Comment (avivt) [inline Lecture5.tex]: would be clearer to separate the exponent of gamma and P, i.e., \gamma^t {P^\pi}^t

Comment (EvZissel) [reply Lecture5.tex]: Hi Aviv @avivt , I've separated the exponent as you suggested

File: Lecture3.tex (hunk 1 above)
Comment (EvZissel) [reply Lecture3.tex]: @avivt I agree - I restored the notation

File: Lecture5.tex (hunk 2 above)
Comment (avivt) [reply Lecture5.tex]: what I meant is \gamma^t {P^\pi}^t instead of (\gamma {P^\pi})^t

Comment (EvZissel) [reply Lecture5.tex]: @avivt I removed the parenthesis: commit ebf9eb5edc1031c1cd049a6d853d3a676adcd7d9

**Comment by avivt** [comment]
Thanks!

**Pull request merged**